docker/waypoint/reset_courses.sql

```diff
@@ -0,0 +1,78 @@
+BEGIN;
+
+DO $$
+DECLARE
+    target_tables text[] := ARRAY[
```

Comment:
Is this something we can either 1) put in an inner folder bc its courses specific, 2) make more broad

Reply:
`docker/waypoint/utils/courses_resetdb.sql`

so in the future if you have other product utils;

`docker/waypoint/utils/pennbasics_add_foodtrucks.sql`